include/ringmesh/mesh/cartesian_grid.h

```diff
@@ -36,10 +36,15 @@
 #pragma once
 
 #include <ringmesh/basic/common.h>
-#include <ringmesh/basic/frame.h>
+#include <ringmesh/basic/frame.h
+#include <ringmesh/geogram_extension/geogram_extension.h>
```

Comment:
You should not need this include. Why do you need this?

Reply:
I used to need it for printing the mesh attributes but I removed the method for the moment